namegen.py

```diff
@@ -0,0 +1,35 @@
+import random
+def main():
+    first_names=('Lydia', 'Emma', 'Andrew', 'Josie', 'Owen', 'Jack', 'Tucker', 'Samantha', 'Frank',
+                'Bill', 'Joey', 'Bob', 'Gorg', 'Jose', 'Jorge', 'Scorge', 'Ben', 'Anita', 'Casey',
+                'Bella', 'Marcus', 'Austin', 'Mat', 'Grace', 'Mia', 'Connor', 'Harrison', 'Charlie',
+                'Taylor', 'Maxine', 'Anja', 'Aaron', 'Erin', 'Ruthie', 'Emily', 'Chloe', 'Matt', 'David',
+                'Kim', 'Cutter', 'Aidan', 'Riley', 'Rosalie', 'Haden', 'Georgina')
+
+    word = ('Lydia', 'Emma', 'Andrew', 'Josie', 'Owen', 'Jack', 'Tucker', 'Samantha', 'Frank',
+                 'Bill', 'Joey', 'Bob', 'Gorg', 'Jose', 'Jorge', 'Scorge', 'Ben', 'Anita', 'Casey',
+                 'Bella', 'Marcus', 'Austin', 'Mat', 'Grace', 'Mia', 'Connor', 'Harrison', 'Charlie',
+                 'Taylor', 'Maxine', 'Anja', 'Aaron', 'Erin', 'Ruthie', 'Emily', 'Chloe', 'Matt', 'David',
+                 'Kim', 'Cutter', 'Aidan', 'Riley', 'Rosalie', 'Haden', 'Georgina')
+first_names='word'[3]
+print first_names
+
+last_name=('Elliott', 'Harrington', 'Bryant', 'Schwieterman', 'Blake', 'Brown', 'Brooks', 'Palma',
```

Comment:
A better name for this variable would be `last_names`, since it contains multiple last names.